build/pipelines/release.yml

```diff
@@ -265,6 +269,231 @@ jobs:
       IndexSources: false
       SymbolServerType: TeamServices
 
+- ${{ if eq(parameters.runCompliance, true) }}:

```

Comment:
Any interest in making this a template or something? I guess the only benefit would be that you're placing most of the compliance stuff under one umbrella.

Reply:
I will pull it into its own template in a future phase. I filed #11974 to remember to clean up the whole release.yml as it's mostly an exported mess from the AzDO pipelines GUI right now.